spec/jobs/state_file/send_reject_resolution_reminder_notification_job_spec.rb

```diff
@@ -110,7 +111,36 @@
               message: message,
               body_args: body_args).exactly(2).times
           end
+        end
+
+        context "when another intake exists with the same hashed SSN" do
+          let!(:other_intake) {
+            create other_intake_class,
+                   efile_submissions: [create(:efile_submission, :accepted)],
+                   primary_first_name: "Mona",
+                   email_address: "monalisa@example.com",
+                   email_address_verified_at: 1.minute.ago,
+                   hashed_ssn: intake.hashed_ssn,
+                   message_tracker: {}
+          }
+
+          context "and which was filed in the same state" do
+            let(:other_intake_class) { :state_file_az_intake }
+
+            it "does not send the message" do
+              expect { described_class.perform_now(intake) }.not_to change(StateFileNotificationEmail, :count)
+            end
           end
+
+          context "and which was filed in another state" do
```

Comment:
@mpidcock was saying that this wasn’t possible?

Reply:
it may indeed not be!

Reply:
on second thought it is still good to test